lib/validation.ts

```
@@ -0,0 +1,96 @@
+			case "email":
+				const emailSchema = z.preprocess(
+					(value) => {
+						if (value === "") return undefined;
+						return value;
+					},
+					field.required
+						? z.email({
+								error: "This field is required",
+							})
+						: z.email().optional()
+				);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)